.github/MODERATOR_APPLICATION.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Community Moderator Request
+about: Request moderator priviledges on a Kubernetes property
+title: 'REQUEST: New moderator for <your-GH-handle>' of '<k8s property>'
```

Comment:
The quotes after <your-GH-handle> and before <k8s property> need to be removed for this to work :grimacing:

Reply:
Also..this file needs to go as `.github/ISSUE_TEMPLATE/moderation_application.md` (just tried it on a dummy repo).